server/app/services/Path.java

```diff
@@ -100,24 +102,47 @@ public Path replacingLastSegment(String finalSegment) {
   }
 
   /**
-   * Append a path to the path.
+   * Append a String path to the path.
```

Comment:
**suggestion:** Add {@link Path} for consistency here as well?

Reply:
Done